v1/transactions/transactions.gunk

```diff
@@ -148,6 +148,12 @@ type GetTransactionRequest struct {
 
 // GetTransactionsRequest is the request message to get a list of transactions.
 type GetTransactionsRequest struct {
+	// UserID is the main identifier of the user
```

Comment:
nit: if possible, please make pb number sequential starts from 1.

Reply:
I think for now this is fine to just change it to 1, and number the fields accordingly since no one is really using it. However, if this is a public API that is already being used, this is correct. In the API definition, please put the pb in order though, so if you're adding UserID and AccountID, add it to the end of the struct instead of on the top.